src/jit/lsra.cpp

```diff
@@ -3007,14 +3225,18 @@ LinearScan::buildRefPositionsForNode(GenTree *tree,
         {
             if (produce != 0)
             {
-                stack->Push(LocationInfo(currentLoc, interval, tree));
+                LocationInfoList list(listNodePool.GetNode(currentLoc, interval, tree));
+                bool added = operandToLocationInfoMap.AddOrUpdate(tree, list);
+                (assert(added), (void)added);
```

Comment:
Instead of a comment you could have a `#define UNUSED_VAR(x) (void)(x)` macro. That should be clear enough to not need a comment.

But which compiler generates a warning in this case? It seems that neither VC++ nor Clang do this.